.github/workflows/update-gems.yml

```diff
@@ -0,0 +1,41 @@
+# TODO abort if there are no changes to the Gemfile.
```

Comment:
I think this probably does abort due to line 23 of update.sh

Reply:
It will yeah, good spot